tfx/components/example_gen/driver.py

```diff
@@ -53,9 +53,10 @@ def update_output_artifact(
     exec_properties: execution properties passed to the example gen.
     output_artifact: the example artifact to be output.
   """
-  output_artifact.custom_properties[
-      utils.FINGERPRINT_PROPERTY_NAME].string_value = str(
-          exec_properties[utils.FINGERPRINT_PROPERTY_NAME])
+  if exec_properties[utils.FINGERPRINT_PROPERTY_NAME] is not None:
```

Comment:
I think we should have a unit test in https://github.com/tensorflow/tfx/blob/master/tfx/components/example_gen/driver_test.py when this is None (or unset)

Do you think you can add that?

Reply:
Done, added tests for the other properties while I was at it.